Source/SpacePort/API/ApiDataFetch.cs

```diff
@@ -75,6 +73,13 @@ public PersonData GetPerson(string name)
             var data = JsonConvert.DeserializeObject<PersonDataResults>(dataResponse.Result.Content);
             return data.results[0];
         }
+
+        public PersonDataResults GetPeople(string name)
```

Comment:
Why is the  method called GetPeople and returns a PersonDataResults? Does not seem to align
Should the method return a List<Person>?